src/server.py

```diff
@@ -78,6 +81,37 @@ def get_date_range(dates):
                 'endDate': max(dates).strftime('%Y-%m-%d %H:%M')}
 
 
+def valve(head_rev, pushlog_id):
+    """Determine which kind of job should been returned"""
+    low_job_return = True
+    COUNTER.increase_the_counter()
+    request_list = []
+    try:
+        request_list = session.query(TaskRequests.head_rev, TaskRequests.pushlog_id,
+                                     TaskRequests.low_job_return).all()
```

Comment:
I am not clear on what this table will hold- can you give an example of what the query will return?  I thought it was a list of revisions and what priority we assigned them- ifthat is the case and we reset the counter every 5th revision, then we don't need much data.